index.html

```diff
@@ -52,8 +62,11 @@ <h1 class="intro">
         <h1 class="works">
             My Recent<br> <span>Works</span>
         </h1>
+        <h1 class="work">
+            My Recent Works
+        </h1>
         <div class="indicator"></div>
-        <section>
+        <section class="grid-container">
```

Comment:
- [ ] Kindly consider the following to ensure a pixel perfect design.

- Your design
<img width="966" alt="image" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/41163571/d30a4192-53bd-4c0b-b6cb-6961db964620">

- Kindly ensure that the screen accommodates three cards, like it is in the original design.
- Kindly increase the spaces  outside the cards.
- Kindly reduce the space between the cards, both horizontally and vertically.